docker-compose.openSearch.yml

```diff
@@ -0,0 +1,50 @@
+version: '2.4'
+services:
+  opensearch:
+    build:
+      context: .
+      dockerfile: Dockerfile-openSearch-dev-
+    ports:
```

Comment:
I don't think we should expose ports of open search by default as it's a backend for the dashboards. Please let me know if there is anything in this container that user should have access to directly (like UI that is not proxies by front end)

Reply:
Yes, You are right. After going into 9200 there is web popup to provide auth credentials. After login you should see some json with basic information. No more information are displayed on port 9200. Therefore we can skip exposing those ports. I've just checked this on my local computer and still works after removing this part.
![Screenshot from 2023-06-19 11-10-40](https://github.com/openimis/openimis-dist_dkr/assets/52816247/eb5e2dc7-c09c-424b-8cec-84e503c9baec)